src/NuGet.Clients/PackageManagement.UI/Resources/Resources.xaml

```diff
@@ -135,27 +135,23 @@
       Property="HorizontalAlignment"
       Value="Center" />
     <Setter
-      Property="StretchDirection"
-      Value="DownOnly" />
+      Property="Height"
+      Value="32" />
     <Setter
-      Property="Stretch"
-      Value="Fill" />
-    <Style.Triggers>
-      <EventTrigger
-        RoutedEvent="ImageFailed">
-        <BeginStoryboard>
-          <Storyboard>
-            <ObjectAnimationUsingKeyFrames
-              Storyboard.TargetProperty="Source"
-              FillBehavior="HoldEnd">
-              <DiscreteObjectKeyFrame
-                KeyTime="0:0:0"
-                Value="{StaticResource BitmapImage_DefaultIcon}">
-              </DiscreteObjectKeyFrame>
-            </ObjectAnimationUsingKeyFrames>
-          </Storyboard>
-        </BeginStoryboard>
-      </EventTrigger>
-    </Style.Triggers>
+      Property="Width"
+      Value="32" />
+    <Setter Property="Source">
+      <Setter.Value>
+        <Binding
+          Path="DataContext.IconUrl"
+          RelativeSource="{RelativeSource Self}"
+          Converter="{StaticResource IconUrlToImageCacheConverter}"
+          ConverterParameter="{x:Static nuget:Images.DefaultPackageIcon}"
+          TargetNullValue="{x:Static nuget:Images.DefaultPackageIcon}"
+          FallbackValue="{x:Static nuget:Images.DefaultPackageIcon}"
+          IsAsync="False"/>
```

Comment:
IsAsync false is default behavior. better to not set values to default values in xaml.